### PR TITLE
Implement a Lobster like feature

### PR DESCRIPTION
**Pull request by BastienDuplessier**
This PR adds a shared links feature. Links are a title and an url.
See below for the implemented view. 
image

### CODE REVIEW

**Comment by gr-im** [comment]
Some pre-nitpicking. 
- I suggest to put the creation form at the begining of the page
- I hope that we will have comments so I think that the feature is still Work In Progress. But I have no issue about merging (after review) a partial version of the feature so I suggest to remove the link in the header. WDYT ?

**Review by gr-im — CHANGES_REQUESTED** [review]
Except for the two suggestion I made early, it is very nice!
Congratulation, it is a very impressive first real contribution!

File: app/views/shared_link_views.ml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          [ img ~a:[ a_class [ "image"; "is-48x48" ] ] ~src ~alt () ]
+      ; td
+          ~a:[ a_class [ "is-vcentered"; "is-fullwidth" ] ]
+          [ a ~a:[ a_href shared_link.url ] [ txt shared_link.title ] ]
+      ; td
+          ~a:[ a_class [ "is-vcentered" ] ]
+          [ span
+              [ txt
+                @@ "le "
+                ^ Templates.Util.format_date shared_link.creation_date
```
Comment (aguillon) [inline app/views/shared_link_views.ml:67]: This prints "le 2023-01-13 06:48", and "le 2023-01-13 **à** 06:48" would be better. It's also true for topics and messages, so I think you can change it safely.

**Review by aguillon — APPROVED** [review]
Very nice, thank you.

It would be nice to have a way of discussing those links in the forum. What do you guys think of adding a shortcut button to create a topic about a link? We could also imagine that clicking the button makes you join an existing topic instead of creating a new one, if it already exists. This may require more stuff in the database though.

This should be added separately of course.

**Review by gr-im — APPROVED** [review]

**Pull request merged**